website_twitter_no_ext_links/models/twitter_feed.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+import base64, json, re, urllib, urllib2
+
+
+class TwitterFeed(models.Model):
+    _name = 'twitter.feed'
+
+    name = fields.Char(
+        string='Feed Name', 
+        help='Just a name to identify this feed',
+        required=True
```

Comment:
develop the habit to add commas at the end of argument lists (and list and tuple entries too). This makes is simple to add stuff without forgetting to add the comma. And a diff will only be one line, not two